types/consts.go

```diff
@@ -7,8 +7,9 @@ func Bool(b bool) *bool { return &b }
 // These exported variables should be treated as constants, to be used in API
 // responses which require *bool fields.
 var (
-	False = Bool(false)
-	True  = Bool(true)
+	False                    = Bool(false)
+	True                     = Bool(true)
+	EthereumZeroAddressBytes = [20]byte{}
```

Comment:
nit: I'd do `EthereumZeroAddress ethcommon.Address`.

This drops the `Bytes`, because it's a byte array, not a byte slice, and "bytes" tends to imply a slice.

you also give it the proper type of Address, which helps avoid misuse, and also allows you to not hard-code the length of 20 here.